CHANGES.rst

```diff
@@ -18,6 +18,7 @@ Bugfixes
   tickets are blocked (:issue:`4242`)
 - Fix event access key prompt not showing when accessing an attachment link
   (:issue:`4255`)
+- Include event title in OpenGraph metadata (:issue:`4288`)
```

Comment:
bugfix or improvement?

Reply:
Thought of it too. Looks like something we overlooked to me, hence a bug.